### ISSUE TITLE
TODO's

### ISSUE THREAD

**Issue by lucashueda** [AI-Unicamp/interspeech_ser#1]
- [x] Data environment setup
- [x] Environment setup
- [x] Run baselines
- [ ] Put txt based logger and/or tensorboard support
- [x] Make a dev set evaluation script isolated to eval on our local validation set
- [ ] Fix output of eval script for categorical task (it is returning logits over 8 classes, it should return a single character classified)
- [ ] Encapsulate metrics calculation
